src/paulscode/android/mupen64plusae/cheat/CheatEditorActivity.java

```diff
@@ -97,7 +95,9 @@ public View getView( int position, View convertView, ViewGroup parent )
     private CheatListAdapter cheatListAdapter = null;
     private AppData mAppData = null;
     private GlobalPrefs mGlobalPrefs = null;
-    private RomHeader mRomHeader = null;
```

Comment:
I tried to move away from creating RomHeader objects all over the place and instead pass around what we need.